Fix a typo in the comment

helunwencser opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #4140

as title

Differential Revision: [D59255153](https://our.internmc.facebook.com/intern/diff/D59255153/)

helunwencser committed:
Fix a typo in the comment

as title

Differential Revision: [D59255153](https://our.internmc.facebook.com/intern/diff/D59255153/)

[ghstack-poisoned]

dbort reviewed (approved):
Thanks for fixing this!

helunwencser committed:
Update on "Fix a typo in the comment"


as title

Differential Revision: [D59255153](https://our.internmc.facebook.com/intern/diff/D59255153/)

[ghstack-poisoned]

helunwencser committed:
Update on "Fix a typo in the comment"


as title

Differential Revision: [D59255153](https://our.internmc.facebook.com/intern/diff/D59255153/)

[ghstack-poisoned]
